pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -23,7 +23,7 @@ import (
 	"k8s.io/klog"
 
 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why was this change made ? I dont see this being used in the patch.

Reply:
It seems Intellij configuration automatically added it. To avoid reverting every time, I would prefer to leave it here.